### PR TITLE
Setup flow and convert existing component files 

### LINKED ISSUE #19: Implement Flow
I really would like to use flow on this project because it helps to scale the application, avoid many bugs and it is more secure to future contributors.

Feel free to others suggests :)

### PR DESCRIPTION
**Pull request by pallymore**
fixes #19

### CODE REVIEW

**Comment by pallymore** [comment]
alternatively, you could also add a babel plugin to convert flow types to prop-types so you will get runtime type checking as well.

https://www.npmjs.com/package/babel-plugin-flow-react-proptypes

File: src/custom/Circle.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/custom/Circle.js]: `import * as React from 'react'`

File: src/custom/Circle.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 
-const Circle = props => {
+type Props = {
+  x: number,
+  y: number,
+  radius: number,
+}
+
+const Circle = (props: Props) => {
```
Comment (dleitee) [inline src/custom/Circle.js]: `const Circle = (props: Props): React.Element<*> => {`

File: src/custom/Rect.js [hunk 3]
```diff
@@ -1,6 +1,15 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/custom/Rect.js]: `import * as React from 'react'`

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
+//@flow
+
 import React, { Component } from 'react'
```
Comment (dleitee) [inline src/index.js]: `import * as React from 'react'`

File: src/stylized/CodeStyle.js [hunk 5]
```diff
@@ -1,7 +1,9 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/stylized/CodeStyle.js]: `import * as React from 'react'`

File: src/stylized/FacebookStyle.js [hunk 6]
```diff
@@ -1,7 +1,9 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/stylized/FacebookStyle.js]: `import * as React from 'react'`

File: src/stylized/InstagramStyle.js [hunk 7]
```diff
@@ -1,9 +1,11 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/stylized/InstagramStyle.js]: `import * as React from 'react'`

File: src/stylized/ListStyle.js [hunk 8]
```diff
@@ -1,7 +1,9 @@
+//@flow
 import React from 'react'
```
Comment (dleitee) [inline src/stylized/ListStyle.js]: `import * as React from 'react'`

File: src/stylized/ListStyle.js [hunk 9]
```diff
@@ -1,7 +1,9 @@
...
 import Wrap from '../Wrap'
+import type { WrapProps } from '../Wrap';
 
-const ListStyle = props => {
+const ListStyle = (props: WrapProps) => {
```
Comment (dleitee) [inline src/stylized/ListStyle.js]: `const ListStyle = (props: WrapProps): React.Element<*> => {`

File: src/stylized/InstagramStyle.js [hunk 10]
```diff
@@ -1,9 +1,11 @@
...
 import Wrap from '../Wrap'
+import type { WrapProps } from '../Wrap';
 
-const InstagramStyle = props => {
+const InstagramStyle = (props: WrapProps) => {
```
Comment (dleitee) [inline src/stylized/InstagramStyle.js]: `const InstagramStyle = (props: WrapProps): React.Element<*> => {`

File: src/stylized/FacebookStyle.js [hunk 11]
```diff
@@ -1,7 +1,9 @@
...
 import Wrap from '../Wrap'
+import type { WrapProps } from '../Wrap';
 
-const FacebookStyle = props => {
+const FacebookStyle = (props: WrapProps) => {
```
Comment (dleitee) [inline src/stylized/FacebookStyle.js]: `const FacebookStyle = (props: WrapProps): React.Element<*> => {`

File: src/stylized/CodeStyle.js [hunk 12]
```diff
@@ -1,7 +1,9 @@
...
 import Wrap from '../Wrap'
+import type { WrapProps } from '../Wrap';
 
-const CodeStyle = props => {
+const CodeStyle = (props: WrapProps) => {
```
Comment (dleitee) [inline src/stylized/CodeStyle.js]: `const CodeStyle = (props: WrapProps): React.Element<*> => {`

File: src/index.js [hunk 13]
```diff
@@ -11,8 +12,37 @@ import ListStyle from './stylized/ListStyle'
...
+  style: {[key: string]: any},
+  type: string,
+  speed: number,
+  width: number,
+  height: number,
+  primaryColor: string,
+  secondaryColor: string,
+}
+
+class ContentLoader extends Component<Props, State> {
```
Comment (dleitee) [inline src/index.js]: class ContentLoader extends React.Component<Props, State> {

**Comment by pallymore** [comment]
updated, thanks @dleitee

**Comment by danilowoz** [comment]
Awesome @pallymore, nice job!! 🎉  
Thanks @dleitee for your review.

**Pull request merged**